src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3667,18 +3667,23 @@
      * <p>For example:</p>
      *
      * <pre>
-     *     A extends Collection
+     *     class MyClass &lt; A extends Collection &gt;
      * </pre>
      *
      * <p>parses as:</p>
      *
      * <pre>
-     * +--TYPE_PARAMETER
-     *     |
-     *     +--IDENT (A)
-     *     +--TYPE_UPPER_BOUNDS
-     *         |
-     *         +--IDENT (Collection)
+     * CLASS_DEF -&gt; CLASS_DEF
+     * |--MODIFIERS -&gt; MODIFIERS
+     * |--LITERAL_CLASS -&gt; class
+     * |--IDENT -&gt; MyClass
+     * |--TYPE_PARAMETERS -&gt; TYPE_PARAMETERS
+     * |   |--GENERIC_START -&gt; &lt;
+     * |   |--TYPE_PARAMETER -&gt; TYPE_PARAMETER
+     * |   |   |--IDENT -&gt; A
+     * |   |   `--TYPE_UPPER_BOUNDS -&gt; extends
+     * |   |       `--IDENT -&gt; Collection
+     * |   `--GENERIC_END -&gt; &gt;
```

Comment:
The first `|` leads to nowhere. Please add `{}` to the example/AST as in https://github.com/checkstyle/checkstyle/commit/107bad1f273adcdf9507dc71ed14b8afa775bbe5